content/swiftui/concepts/drag-and-drop/drag-and-drop.md

```diff
@@ -0,0 +1,155 @@
+---
+Title: 'drag-and-drop' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```

Comment:
```suggestion
Title: 'Drag and Drop' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
```